### ISSUE TITLE
when using svg symbol, an unwanted white "disc" created behind the icon

### ISSUE THREAD

**Issue by PchanAUS** [andreaordonselli/qgis2o.gis#28]
![Screenshot 2024-12-04 185452](https://github.com/user-attachments/assets/41193238-5c92-4f53-8e76-012e3397433c)

thx Andrea, great work!

**Comment by andreaordonselli** [comment]
Hi, I set the circle in the back of the icons to see them better on the screen, you just delete this part of code in the exported style or in the olStyleScripts.py file (if you want to make circle deletion default).:

```javascript
new ol.style.RegularShape({
              points: 20,
              radius: %(radius)s,
              fill: new ol.style.Fill({
              color: 'white'
                 }),
              stroke: new ol.style.Stroke({
                color: "black",
                width: 1.5
                })
               })
             }),
```

**Comment by PchanAUS** [comment]
Thx Andrea, it works great.
pls keep up with the great work!

**Comment by andreaordonselli** [comment]
This issue has been resolved in the [v2.26.0](https://github.com/andreaordonselli/qgis2o.gis/releases/tag/v2.25.0) release
Thanks for your report
